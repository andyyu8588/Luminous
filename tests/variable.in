benbiensur = "POG";
print(benbiensur);
benbiensur = benbiensur + " ANDY";
print(benbiensur);
benbiensur = 3.1415926;
print(benbiensur);
benbiensur = false;
print(benbiensur);
benbiensur = 1 < 2;
print(benbiensur);
benbiensur2 = 3.14 * 99.123;
print(benbiensur2);