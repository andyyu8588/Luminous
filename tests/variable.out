POG
POG ANDY
3.1415926
false
true
311.24622